### DISCUSSION TITLE
Unable to install cs1.6 app 90 branch -beta betapassword Steam_Legacy ./csserver install error with bad betapassword

### DISCUSSION THREAD

**Discussion by yunodeb** [GameServerManagers/LinuxGSM#4533] [Q&A]
Hi there,

Thanks for this amazing tool!

There are lot of issue with the hlds lastest build 9907, metamod and amxmodx have to be updated to running fine.

The better way is to install the last build before the last update, and we have to use the -beta branch 90 with betapassword steam_legacy. 

I got an issue when i try to install it with "./csserver install"

Linuxgsm returns an error because of "bad betapassword".

I dont understand, i've used this lines in my csserver.cfg :

## SteamCMD Settings | https://docs.linuxgsm.com/steamcmd
# Server appid
appid="90"
steamcmdforcewindows="no"
# SteamCMD Branch | https://docs.linuxgsm.com/steamcmd/branch
branch="-beta"
betapassword="Steam_Legacy"
# Master Server | https://docs.linuxgsm.com/steamcmd/steam-master-server
steammaster="true"

Any idea?
Thanks

**Comment by sgloutnikov** [comment]
The branch is just `steam_legacy`. Adding `branch="steam_legacy"` to my `common.cfg` worked, but not with adding it in `instance.cfg`.
